Core/include/Acts/Geometry/GeometryIdentifier.hpp

```diff
@@ -96,6 +114,8 @@ class GeometryIdentifier {
   // clang-format on
 
   Value m_value = 0;
+  double m_misalignmentX = 0.0;
+  double m_misalignmentY = 0.0;
```

Comment:
Same comment as on #2291, we can't change `GeometryIdentifier` to accommodate this information.